metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -1446,6 +1453,15 @@ public List<Pair<String, RecordTemplate>> generateDefaultAspectsIfMissing(@Nonnu
       }
     }
 
+    if (shouldCheckBrowsePathV2 && latestAspects.get(BROWSE_PATHS_V2_ASPECT_NAME) == null) {
+      try {
+        BrowsePathsV2 generatedBrowsePathV2 = buildDefaultBrowsePathV2(urn, false);
```

Comment:
should we NOT generate browse Path v1 if v2 is enabled? maybe not for now, but later probably yes

Reply:
yeah right now we're still generating both - my thinking was for flexibility so that we can flip feature flags one way or the other. but eventually we could stop producing based on flag or stop producing old ones entirely